.travis.yml

```diff
@@ -162,6 +162,9 @@ install:
       sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main';
       sudo apt update;
       sudo apt install cmake;
+      sudo mkdir /usr/local/cmake-latest/bin
```

Comment:
This works for me:
```
apt:
    sources:
      - sourceline: 'deb https://apt.kitware.com/ubuntu/ bionic main'
        key_url: 'https://apt.kitware.com/keys/kitware-archive-latest.asc'

before_install:
  - |
    if [ "$TRAVIS_OS_NAME" = "linux" ]; then
      # CMake from Kitware is installed in /usr/bin
      # TravisCI installs its own cmake to another location which overrides other installations
      # if they don't call the new binary directly
      alias cmake="/usr/bin/cmake"
    fi
  - |
```

Reply:
@zeule 

Thanks! However, we will have to wait to see the results, since Travis is so slow and only does one build at a time...

Reply:
@zeule 
Sadly, your method did not work: https://travis-ci.com/github/arvidn/libtorrent/jobs/400759993#L1323. Thanks anyway. I will now wait for the results of v3.

Reply:
Why did it pool 3.16.8 from the kitware repo? In any case, that alias command it enough, you don't need to mess with directories.

Reply:
@zeule 

> Why did it pull 3.16.8 from the kitware repo? In any case, that alias command it enough, you don't need to mess with directories.

It didn't, it pulled 3.18.4: https://travis-ci.com/github/arvidn/libtorrent/jobs/400759993#L1218

Reply:
Ah, yes. I was wrong. Could you add, please, `alias` call (without parameters) to check that the alias was created?

Reply:
@zeule I'll do that after getting a result from https://travis-ci.com/github/arvidn/libtorrent/jobs/400784318. I screwed up the `mkdir` invocation (missed the `-p`) so yeah...

Reply:
The initial method worked out: https://travis-ci.com/github/arvidn/libtorrent/jobs/400784318. Not sure why yours failed, but this is good enough for me.